andela/travel_tool_front
# # 160710694  Create guest house

#### What does this PR do?
 - A travel admin can add a guesthouse 


#### Description of Task to be completed?
 - Create a UI/UX for the travel admin to add a guesthouse
 - Add the functionality for creating a guesthouse


#### How should this be manually tested?
- Clone the repository and change directory into the folder `cd travel_tool_front`
- Checkout from develop to this branch and pull the branch
- Start the application using `make start`
- Login to the application and click on the accommodation tab in the left sidebar
- You should see a add guest house button 
- Click on it and a form to add a new guest house will pop up

#### Any background context you want to provide?
- You have to set your role to travel admin in other to access the accommodation pages
- cloudinary is used to upload images, make sure you have a cloudinary account and have unsigned upload enable.
- set your cloudinary details in the .env file with the following key `REACT_APP_CLOUNDINARY_API='Your cloudinary URL'` `REACT_APP_PRESET_NAME= Name of the preset for unsigned uplaod`


#### What are the relevant pivotal tracker stories?

- [160710694](https://www.pivotaltracker.com/story/show/160710694)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/20597418/46426559-d6296a80-c736-11e8-9b22-0c7a3dd83e71.png)


#### Questions:
